PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -1125,6 +1122,19 @@ def __init__(self, path=None, excludes=(), replace_paths=(), implies=(), graph=N
         self._deferred_modules = deque()
         self._initialized = False
 
+    def msg(self, level, s, *args):
+        """
+        Print a debug message with the given level
+        """
+        if s:
+            message = "%s %s" % (s, ' '.join(map(repr, args)))
+            if level <= 1:
+                logger.warning(message)
+            elif level <= 2:
+                logger.info(message)
+            elif level <= 3:
+                logger.debug(message)
```

Comment:
Logging prints out many messages with the level set to 3. I made a judgement call that raising the level to 4 would print out too many messages. My position here is indefensible, so feel free to change the numbers if you want.